scripts/levelManagement.js

```diff
@@ -0,0 +1,136 @@
+function configureDownloadLevelButton() {
+    var downloadLevelButton = document.getElementById(
+        "selected-level-download"
+    );
+    refreshElement(downloadLevelButton);
+    var downloadLevelButton = document.getElementById(
+        "selected-level-download"
+    );
+    downloadLevelButton.addEventListener("click", (e) => {
+        var level = document.getElementById("selected-level").innerHTML;
+        var url = `/cgi-bin/downloadLevel.py?level-name=${level}`;
+        var request = new XMLHttpRequest();
+        request.open("GET", url, true);
+        request.onload = function () {
+            var res = JSON.parse(request.responseText);
+            if (res["success"] === "true") {
+                window.open(res["url"]);
+            }
+        };
+        request.onerror = function () {};
+        request.send(); // create FormData from form that triggered event
+        event.preventDefault();
+    });
+}
+
+function configureDeleteLevelButton() {
+    var deleteLevelButton = document.getElementById("selected-level-delete");
+    refreshElement(deleteLevelButton);
+    var deleteLevelButton = document.getElementById("selected-level-delete");
+    deleteLevelButton.addEventListener("click", (e) => {
+        var level = document.getElementById("selected-level").innerHTML;
+        var url = `/cgi-bin/deleteLevel.py?level-name=${level}`;
+        var request = new XMLHttpRequest();
+        request.open("GET", url, true);
+        request.onload = function () {
+            var res = JSON.parse(request.responseText);
+            if (res["success"] === "true") {
+                updateLevelModule();
+            }
+        };
+        request.onerror = function () {};
+        request.send(); // create FormData from form that triggered event
+        event.preventDefault();
+    });
+}
+
+function getSelectedLevel(responseJSON) {
+    var level = JSON.parse(responseJSON)["level-name"];
+    document.getElementById("selected-level").innerHTML = level;
+    configureDownloadLevelButton();
+    configureDeleteLevelButton();
+}
+
+function getAvailableLevels(responseJSON) {
```

Comment:
The same here. This function modifies some elements and don't return a value.

Reply:
Could you specify the issue? I don't see a problem with not returning anything under the current implementation.

Reply:
This functions edit some elements of the UI. So I think that it is better a name like ```updateAvailableLevels``` or something like that.